Firmware/LowLevel/lib/JY901_SERIAL/JY901.cpp

```diff
@@ -10,26 +10,23 @@ CJY901 ::CJY901(SerialPIO *serial)
 	this->serial = serial;
 }
 
-void CJY901::begin(int baudrate) {
+void CJY901::begin(unsigned long baudrate) {
 	serial->begin(baudrate);
 	delay(1000);
-	uint8_t unlock[] = {0xFF,0xF0,0xF0,0xF0,0xF0};
+	/*uint8_t unlock[] = {0xFF,0xF0,0xF0,0xF0,0xF0}; // undocumented / unknown?

```

Comment:
Couldn't find this unlock code in any documentation. Only in a doubtful Amazon comment. So I disabled it.

Reply:
https://forum.arduino.cc/t/help-calibrating-witmotion-inclinometer-through-ttl-with-mega2560/941394/1
This is what my Google-fu gave me, not sure if it is related thou

Reply:
it is needed I remember sniffing the configuration of the GUI app to see how they do it because just sending config didn't work

Reply:
Thanks for your comments!
After reading @FadeFx comment and link I immediately thought: "Could be an Auto-Baud detection sequence", but in-between tested it. It's not :-)
Will uncomment it again.